src/sage/rings/qqbar.py

```diff
@@ -118,6 +118,9 @@
     sage: QQbar((-8)^(1/3))
     1.000000000000000? + 1.732050807568878?*I
     sage: AA((-8)^(1/3))
+    doctest:warning...
+    DeprecationWarning: Raising negative algebraic real number will give principal root in future
+    See https://github.com/sagemath/sage/issues/38362 for details.
     -2
```

Comment:
You omitted a period at the end here.

But I suggested to you a broken sentence: "Raising negative algebraic real number will give principal root in future." Sorry!

A good sentence is "Taking the root of an algebraic real number will yield the principal root in the future."